### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by laraconda**
## Summary of the changes in this pull request

It was not possible to start the program via `python3 rednotebook/journal.py`. Now it is possible after reverting the file jorunal.py (as suggested here: https://github.com/jendrikseipp/rednotebook/pull/663/files#r1064037003)

## Pull request checklist

- [ ] I have added an entry in `CHANGELOG.md` including my name and issue and/or pull request number.
- [ ] If applicable: I have removed the corresponding entry in `TODO.md`.

### CODE REVIEW

File: rednotebook/gui/options.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def get_value(self):
 
     def set(self):
         """Apply the current setting"""
-        if selected := self.get_value():
+        if self.get_value():
```
Comment (laraconda) [inline rednotebook/gui/options.py:89]: This one was not part of the comments in #663 but the style checker flagged it.

**Comment by jendrikseipp** [comment]
I started the CI tests now. Ping me once this is ready for review.

**Comment by laraconda** [comment]
Its ready. The only problem is the style checker finds a problem: _rednotebook/gui/clouds.py:149:111: E501 line too long (117 > 110 characters)_
That line corresponds to the html code in the cloud maker.
I suppose the best way to handle this is to tell the checker to ignore it. Is there a way to do it?

File: rednotebook/gui/clouds.py [hunk 2]
```diff
@@ -138,20 +138,15 @@ def _get_cloud_body(self, cloud_words):
...
         for word, count in cloud_words:
             font_factor = (count - min_count) / delta_count
             font_size = int(min_font_size + font_factor * font_delta)
 
             # Add some whitespace to separate words
             html_elements.append(
-                '<a href="/#search-%s">'
-                '<span style="font-size:%spx">%s</span></a>&#160;'
-                % (self.link_index, font_size, word)
+                f'<a href="/#search-{self.link_index}"><span style="font-size:{font_size}px">{word}</span></a>&#160;'
```
Comment (jendrikseipp) [inline rednotebook/gui/clouds.py:149]:
```suggestion
                f'<a href="/#search-{self.link_index}"><span '
                f'style="font-size:{font_size}px">{word}</span></a>&#160;'
```

Comment (laraconda) [reply rednotebook/gui/clouds.py:149]: Oh, that's a nice way to break it.

**Comment by laraconda** [comment]
I amended the previous commit to add the line break.

**Pull request merged**